samples/client/petstore-security-test/java/okhttp-gson/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -169,11 +169,22 @@ public OkHttpClient getHttpClient() {
     /**
      * Set HTTP client
      *
-     * @param httpClient An instance of OkHttpClient
+     * @param newHttpClient An instance of OkHttpClient
      * @return Api Client
      */
-    public ApiClient setHttpClient(OkHttpClient httpClient) {
-        this.httpClient = httpClient.newBuilder().addInterceptor(getProgressInterceptor()).build();
+     public ApiClient setHttpClient(OkHttpClient newHttpClient) {
```

Comment:
@burberius looks like an additional space was added to this line. Can you please remove it to make the indentation 4-space?